api-service/service/hosts.go

```diff
@@ -275,7 +275,7 @@ func (as *APIService) ListEnvironments(location string, environment string, olde
 // DismissHost dismiss the specified host
 func (as *APIService) DismissHost(hostname string) error {
 	filter := dto.AlertsFilter{OtherInfo: map[string]interface{}{"hostname": hostname}}
-	if err := as.AckAlerts(filter); err != nil {
+	if err := as.RemoveAlertsNODATA(filter); err != nil {
```

Comment:
Are you sure you should replace AckAlerts with RemoveAlertsNODATA?
This are two different things!
Should we add the RemoveAlertsNODATA part only?